src/stan/lang/function_signatures.h

```diff
@@ -516,6 +516,62 @@ add("gaussian_dlm_obs_lpdf", bare_expr_type(double_type()), bare_expr_type(matri
+add("gp_exponential_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)),
+    bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_exponential_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(bare_array_type(double_type(), 1)));
+add("gp_exponential_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)),
+    bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(bare_array_type(double_type(), 1)));
+
+add("gp_matern52_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(double_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern52_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(double_type(), 1)),
+    bare_expr_type(bare_array_type(double_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern52_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern52_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)),
+    bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern52_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(bare_array_type(double_type(), 1)));
+add("gp_matern52_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)),
+    bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(bare_array_type(double_type(), 1)));
+
+add("gp_matern32_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(double_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern32_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(double_type(), 1)),
+    bare_expr_type(bare_array_type(double_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern32_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern32_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)),
+    bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(double_type()));
+add("gp_matern32_cov", bare_expr_type(matrix_type()), bare_expr_type(bare_array_type(vector_type(), 1)), bare_expr_type(double_type()), bare_expr_type(bare_array_type(double_type(), 1)));
```

Comment:
Can you break all of these lines at the same place so I can try to visually track what's changing between each line? Thanks!